fix(jans-auth-server): fix build after dependencies upgrade (tika)

**Describe the bug**

After dependencies upgrade we got build failure due to wrong imports
```
11:45:52,011 [ERROR] COMPILATION ERROR : 
11:45:52,011 [INFO] -------------------------------------------------------------
11:45:52,011 [ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/main/java/io/jans/as/server/token/ws/rs/TxTokenValidator.java:[23,29] package org.apache.tika.utils does not exist
11:45:52,012 [ERROR] /home/jetty/.jenkins/workspace/jans-auth-server/jans-auth-server/server/src/main/java/io/jans/as/server/service/net/UriService.java:[9,29] package org.apache.tika.utils does not exist
```

--- a/jans-auth-server/server/src/main/java/io/jans/as/server/service/net/UriService.java
+++ b/jans-auth-server/server/src/main/java/io/jans/as/server/service/net/UriService.java
@@ -6,7 +6,7 @@ import io.jans.as.model.util.URLPatternList;
 import jakarta.ejb.Stateless;
 import jakarta.inject.Inject;
 import jakarta.inject.Named;
-import org.apache.tika.utils.StringUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.json.JSONObject;
 import org.slf4j.Logger;
 

--- a/jans-auth-server/server/src/main/java/io/jans/as/server/token/ws/rs/TxTokenValidator.java
+++ b/jans-auth-server/server/src/main/java/io/jans/as/server/token/ws/rs/TxTokenValidator.java
@@ -20,7 +20,7 @@ import jakarta.inject.Named;
 import jakarta.ws.rs.WebApplicationException;
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
-import org.apache.tika.utils.StringUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 
 /**

